Microsoft.Identity.Web/Resource/ScopesRequiredByWebAPIExtension.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Security.Claims;
+using System.Text;
+
+namespace Microsoft.Identity.Web.Resource
+{
+    public static class ScopesRequiredByWebAPIExtension
+    {
+        /// <summary>
+        /// When applied to a Controller, verifies that the user authenticated in the Web API has any of the
```

Comment:
Comment should describe that this method throws an HttpRequestException with a 401 code in case verification fails.

Reply:
Done